server/src/test/java/org/elasticsearch/index/mapper/DateFieldMapperTests.java

```diff
@@ -100,20 +66,12 @@ public void testDefaults() throws Exception {
     }
 
     public void testNotIndexed() throws Exception {
-        String mapping = Strings.toString(XContentFactory.jsonBuilder().startObject().startObject("type")
-                .startObject("properties").startObject("field").field("type", "date").field("index", false).endObject().endObject()
-                .endObject().endObject());
 
-        DocumentMapper mapper = parser.parse("type", new CompressedXContent(mapping));
+        DocumentMapper mapper = createDocumentMapper(fieldMapping(b -> b
```

Comment:
I've been running the auto-formatter over chunks like this while I go because one day that auto-formatter will run on our whole code base and I figure if I grab stuff like this now it'll make the diff smaller. And I can look at the results and make sure it does a good job.